test/util/server/server.go

```diff
@@ -607,18 +608,24 @@ func WaitForServiceAccounts(clientset kclientset.Interface, namespace string, ac
 
 // CreateNewProject creates a new project using the clusterAdminClient, then gets a token for the adminUser and returns
 // back a client for the admin user
-func CreateNewProject(clusterAdminClient *client.Client, clientConfig restclient.Config, projectName, adminUser string) (kclientset.Interface, *client.Client, error) {
+func CreateNewProject(clusterAdminClient *client.Client, clientConfig restclient.Config, projectName, adminUser string) (kclientset.Interface, *client.Client, *restclient.Config, error) {
+	authorizationClient, err := authorizationclient.NewForConfig(&clientConfig)
```

Comment:
@mfojtik did we solve this two different ways?  this prevents a proliferation of different client return values from the various helpers.